.ci/format.sh

```diff
@@ -35,13 +35,14 @@ format() {
         src/Initializer/time_stepping/GlobalTimestep.cpp
         src/Initializer/tree/LTSSync.hpp
         src/Kernels/common.hpp
+        src/Kernels/DynamicRupture.cpp
         src/Kernels/PointSourceCluster.h
         src/Kernels/PointSourceClusterOnHost.h
         src/Kernels/PointSourceClusterOnHost.cpp
         src/Kernels/PointSourceClusterOnDevice.h
         src/Kernels/PointSourceClusterOnDevice.cpp
-	src/Kernels/Receiver.h
-	src/Kernels/Receiver.cpp
+	    src/Kernels/Receiver.h
+	    src/Kernels/Receiver.cpp
```

Comment:
Please fix indendation here.

Reply:
Weirdly, the VS code shows the correct indentation, but it is off on Git Hub. But I fixed it, nevertheless. Thank you

Reply:
Maybe a tab/spaces mismatch?